packages/hooks/src/stories/ConnectWallet.stories.tsx

```diff
@@ -1,19 +1,26 @@
-import React from 'react';
+import React, { useEffect } from 'react';
 import { Button } from '@chakra-ui/react';
-import { Provider, useWallet } from '..';
+import { NETWORKS, Provider, useWallet } from '..';
 
 export default {
   title: 'Hooks/useWallet',
 };
 
 const DefaultUsingProvider = () => {
-  const { connection, connectWallet, disconnectWallet, connected } = useWallet();
+  const { connection, connectWallet, disconnectWallet, connected, correctNetwork } = useWallet();
+
+  useEffect(() => {
```

Comment:
Any reason your using a named import here and we use the default React import everywhere else?

Just for my understanding, I personally find it confusing if both are used.

Reply:
Are you referring to using `useEffect` instead of `React.useEffect`?

Reply:
Oh yeah that's just me being dumb. 😬 I'll fix that